include/lsst/afw/geom/SkyWcs.h

```diff
@@ -48,9 +50,9 @@ Make a WCS CD matrix
 
 @param[in] scale  Pixel scale as an angle on sky/pixels
 @param[in] orientation  Position angle of focal plane +Y, measured from N through E.
-                        At 0 degrees, +Y is along N and +X is along E/W if flipX false/true
-                        At 90 degrees, +Y is along E and +X is along S/N if flipX false/true
-@param[in] flipX  Fip x axis (see orientation for details)?
+                        At 0 degrees, +Y is along N and +X is along W/E if flipX false/true
+                        At 90 degrees, +Y is along E and +X is along N/S if flipX false/true
+@param[in] flipX  Fip x axis? See orientation for details.
```

Comment:
I changed it because the `afw::image::Wcs` has an `isFlipped` method which I needed to port to `SkyWcs`. As I ported it I realized it had the opposite sign of this argument, so I modified this code to match, for backwards compatibility. Fortunately very little code actually cares about the sign; the only place that I know of where it matters is a bit of code `meas_extensions_astrometryNet`.